src/UhdmAst.cpp

```diff
@@ -860,12 +869,16 @@ namespace UhdmAst {
 
           // Get parameter assignments
           itr = vpi_iterate(vpiParamAssign, obj_h);
+          std::map<std::string, AstNode*> parameter_map;
```

Comment:
Does this need to be a map? Seems like a set. It's fine if it's going to be used like a map in the future.

Reply:
It _was_ used at one point to detect whether a parameter is overridden, but their values turned out to be consistent. Changed to a set.